feature/request-money/src/main/kotlin/org/mifospay/feature/request/money/ShowQrContent.kt

```diff
@@ -10,94 +10,123 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.width
-import androidx.compose.material3.HorizontalDivider
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import coil.compose.AsyncImage
 import org.mifospay.core.designsystem.component.MifosButton
-import java.util.Locale
+import org.mifospay.core.designsystem.component.MifosOutlinedButton
 
 @Composable
 internal fun ShowQrContent(
     qrDataBitmap: Bitmap,
-    qrDataString: String,
+//    qrDataString: String, // this parameter is not used in new figma design
     showAmountDialog: () -> Unit,
     modifier: Modifier = Modifier,
-    amount: String? = null,
+//    amount: String? = null, // this parameter is not used in new figma design
+    onShare: () -> Unit,
 ) {
-    Column(
-        modifier = modifier.fillMaxWidth(),
-        horizontalAlignment = Alignment.CenterHorizontally,
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .background(
+                getColor(),
+            )
+            .padding(25.dp),
     ) {
-        AsyncImage(
-            model = qrDataBitmap,
-            contentDescription = null,
-            modifier = Modifier
-                .padding(20.dp)
-                .weight(1f),
-        )
-
-        Spacer(modifier = Modifier.height(12.dp))
-
-        MifosButton(
-            modifier = Modifier
-                .width(150.dp),
-            onClick = { showAmountDialog() },
-        ) {
-            Text(text = stringResource(id = R.string.feature_request_money_set_amount))
-        }
-
-        Spacer(modifier = Modifier.height(12.dp))
-
-        HorizontalDivider()
-
         Column(
             modifier = Modifier
-                .fillMaxWidth()
-                .padding(20.dp),
-            horizontalAlignment = Alignment.Start,
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.SpaceBetween,
+            horizontalAlignment = Alignment.CenterHorizontally,
```

Comment:
enable vertical scrolling.

Reply:
The content of screen is not overflowing

Reply:
@Nagarjuna0033 Try rotating your phone.